jep/0000/README.adoc

```diff
@@ -0,0 +1,285 @@
+
+|===
+
+
+== Abstract
+
+As the Jenkins community has grown, there have been ad hoc groups sprouting up
+to tackle specific areas of focus within the project. This proposal describes a
+"Special Interest Group" (SIG) model to provide structure so that these groups can
+become more formalized, with consistent expectations of their operation, to
+help new contributors join areas of effort which align with their interests.
+
+[[specification]]
+== Specification
+
+Much of this specification is modeled after the Kubernetes project's "SIG"
+model. footnoteref:[sig-governance, https://github.com/kubernetes/community/blob/master/sig-governance.md]
+
+
+In order to standardize Special Interest Group efforts, create maximum
+transparency, and route contributors to the appropriate SIG, SIGs should follow
+the guidelines stated below:
+
+* Meet regularly, at least for 30 minutes every 3 weeks, except November and December
+* Keep up-to-date meeting notes, somewhere publicly linked from the SIG's page on jenkins.io
+* Announce meeting agenda and minutes after each meeting, on their SIG mailing list
+* Record SIG meeting, such as via <<video, YouTube Live Streaming>>, and make
+  it publicly available. Text-based meetings can use `#jenkins-meeting` on
+  Freenode..
+* Ensure the SIG's mailing list, and optional chat channel are archived
+* Report activity in the weekly community meeting at least once every 6 weeks
+* Participate in release planning meetings and retrospectives, and burndown
+  meetings, as needed.
+* Ensure related work happens in a project-owned GitHub org and repository,
+  with code and tests explicitly owned and supported by the SIG, including issue
+  triage, PR reviews, test-failure response, bug fixes, etc.
+* Use the above forums as the primary means of working, communicating, and
+  collaborating, as opposed to private emails and meetings
+
+
+All Special Interest Groups are subject to the Jenkins project
+link:https://jenkins.io/conduct[Code of Conduct].
+
+=== Roles
+
+- **SIG Participant**: active in one or more areas of the project; wide
+  variety of roles are represented
+- **SIG Lead**: SIG organizer
+
+=== Creation and Maintenance
+
+==== Prerequisites
+
+* Propose the new SIG publicly, including a brief mission statement, by
+  emailing jenkinsci-dev@googlegroups.com and jenkinsci-users@googlegroups.com,
+  then wait a couple of days for feedback
+* Organize meetings using video or text as needed. No need to wait for the a
+  regularly scheduled Jenkins Governance meeting to discuss. Please report
+  summary of ad hoc SIG meetings to the the SIG mailing list.
+* Use existing proposal and <<pull-request, PR process>>
+* Announce new SIG on jenkinsci-dev@googlegroups.com
+* Submit a PR to add a row for the SIG to the table in the kubernetes/community README.md file, to create a kubernetes/community directory, and to add any SIG-related docs, schedules, roadmaps, etc. to your new kubernetes/community/SIG-foo directory.
```

Comment:
I guess you forgot to update this with Jenkins related values  when copying https://github.com/kubernetes/community/blob/master/sig-governance.md